From 248af23b2b5ee226321df52309e0094da9e6434a Mon Sep 17 00:00:00 2001
From: MichaIng <micha@dietpi.com>
Date: Mon, 23 Nov 2020 16:17:31 +0100
Subject: [PATCH 0413/2201] configs: Enable WireGuard kernel module on armhf
 configs

CONFIG_WIREGUARD=m implies CONFIG_ARM_CRYPTO=y, hence the latter is removed.

Signed-off-by: MichaIng <micha@dietpi.com>
---
 arch/arm/configs/bcm2709_defconfig | 2 +-
 arch/arm/configs/bcm2711_defconfig | 2 +-
 arch/arm/configs/bcmrpi_defconfig  | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index 30221357123a..6d5c83278692 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -57,7 +57,6 @@ CONFIG_KERNEL_MODE_NEON=y
 # CONFIG_SUSPEND is not set
 CONFIG_PM=y
 CONFIG_RASPBERRYPI_FIRMWARE=y
-CONFIG_ARM_CRYPTO=y
 CONFIG_CRYPTO_SHA1_ARM_NEON=m
 CONFIG_CRYPTO_AES_ARM=m
 CONFIG_CRYPTO_AES_ARM_BS=m
@@ -481,6 +480,7 @@ CONFIG_DM_DELAY=m
 CONFIG_NETDEVICES=y
 CONFIG_BONDING=m
 CONFIG_DUMMY=m
+CONFIG_WIREGUARD=m
 CONFIG_IFB=m
 CONFIG_MACVLAN=m
 CONFIG_IPVLAN=m
diff --git a/arch/arm/configs/bcm2711_defconfig b/arch/arm/configs/bcm2711_defconfig
index bb43adf957a9..fcc21e2feaee 100644
--- a/arch/arm/configs/bcm2711_defconfig
+++ b/arch/arm/configs/bcm2711_defconfig
@@ -57,7 +57,6 @@ CONFIG_KERNEL_MODE_NEON=y
 # CONFIG_SUSPEND is not set
 CONFIG_PM=y
 CONFIG_RASPBERRYPI_FIRMWARE=y
-CONFIG_ARM_CRYPTO=y
 CONFIG_CRYPTO_SHA1_ARM_NEON=m
 CONFIG_CRYPTO_AES_ARM=m
 CONFIG_CRYPTO_AES_ARM_BS=m
@@ -488,6 +487,7 @@ CONFIG_DM_DELAY=m
 CONFIG_NETDEVICES=y
 CONFIG_BONDING=m
 CONFIG_DUMMY=m
+CONFIG_WIREGUARD=m
 CONFIG_IFB=m
 CONFIG_MACVLAN=m
 CONFIG_IPVLAN=m
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index d8484c28caed..77c400a1ecd9 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -52,7 +52,6 @@ CONFIG_VFP=y
 # CONFIG_SUSPEND is not set
 CONFIG_PM=y
 CONFIG_RASPBERRYPI_FIRMWARE=y
-CONFIG_ARM_CRYPTO=y
 CONFIG_CRYPTO_SHA1_ARM=m
 CONFIG_CRYPTO_AES_ARM=m
 CONFIG_OPROFILE=m
@@ -475,6 +474,7 @@ CONFIG_DM_DELAY=m
 CONFIG_NETDEVICES=y
 CONFIG_BONDING=m
 CONFIG_DUMMY=m
+CONFIG_WIREGUARD=m
 CONFIG_IFB=m
 CONFIG_MACVLAN=m
 CONFIG_IPVLAN=m
-- 
2.17.1

